android/java/org/chromium/chrome/browser/onboarding/OnboardingPrefManager.java

```diff
@@ -194,7 +194,6 @@ public void setOnboardingShownForSkip(boolean isShown) {
     public boolean showOnboardingForSkip(Context context) {
         boolean shouldShow = PackageUtils.isFirstInstall(context)
                              && !hasOnboardingShownForSkip()
-                             && (ChromeFeatureList.isEnabled(BraveFeatureList.BRAVE_REWARDS) && !UserPrefs.get(Profile.getLastUsedRegularProfile()).getBoolean(BravePref.ENABLED))
```

Comment:
are we fine with x86 devices? as this check `ChromeFeatureList.isEnabled(BraveFeatureList.BRAVE_REWARDS)` was to exclude them as no rewards there.

Reply:
good catch. i forgot about x86 devices. i will revert it.